addons/website_blog/views/website_blog_templates.xml

```diff
@@ -302,7 +302,7 @@
                     <div class="media-body">
                         <input name="blog_post_id" t-att-value="blog_post.id" type="hidden"/>
                         <textarea rows="3" name="comment" class="form-control" placeholder="Write a comment..."></textarea>
-                        <button type="submit" class="btn btn-primary mt8">Post</button>
+                        <button type="submit" id="btn-loading" data-loading-text="Posting..." class="btn btn-primary mt8">Post</button>
```

Comment:
I don"t think the "Posting ...." will be translate if it is store in a data attribute.